actionview/actionview.gemspec

```diff
@@ -36,7 +36,7 @@ Gem::Specification.new do |s|
   s.add_dependency "activesupport", version
 
   s.add_dependency "builder",       "~> 3.1"
-  s.add_dependency "erubi",         "~> 1.4"
+  s.add_dependency "erubi",         "~> 1.11"
```

Comment:
`erubi` `1.11` is out, which makes all this simpler. I don't expect bumping the minimum requirement to be an issue. Could it?